src/test/java/dev/dokan/dokan_java/examples/DirListingFSTest.java

```diff
@@ -4,24 +4,14 @@
 import dev.dokan.dokan_java.constants.dokany.MountOption;
 import dev.dokan.dokan_java.constants.microsoft.FileSystemFlag;
 import dev.dokan.dokan_java.structure.EnumIntegerSet;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.slf4j.impl.SimpleLogger;
+
 
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
 public class DirListingFSTest {
 
-    public static final Logger LOG = LoggerFactory.getLogger(DirListingFSTest.class);
-
-    static {
-        System.setProperty(SimpleLogger.DEFAULT_LOG_LEVEL_KEY, "info");
-        System.setProperty(SimpleLogger.LOG_FILE_KEY, "System.out");
-        System.setProperty(SimpleLogger.SHOW_DATE_TIME_KEY, "true");
-        System.setProperty(SimpleLogger.DATE_TIME_FORMAT_KEY, "HH:mm:ss:SSS");
-    }
```

Comment:
Why removing the logger dependecy? This is an example and with logging the user can see what happen. I don't think it it should be removed.

Reply:
We can keep the SLF4J for all tests, no problem with that. I should probably correct this